app/controllers/api/services_controller.rb

```diff
@@ -14,6 +15,15 @@ class Api::ServicesController < Api::BaseController
     actions.sublayout 'api/service'
   end
 
+  def index
```

Comment:
I see many tips (in the logs) to improve the performance.

![image](https://user-images.githubusercontent.com/11318903/95969577-34ea4a00-0e0f-11eb-9d27-2a4bc9c167cb.png)

Reply:
For a following PR 👍